design docs/durandal play by play.txt

```diff
@@ -0,0 +1,55 @@
+Gameplay Example
+
+Player1 joins the server and enters the customization screen.
+
+They choose a government entity with specializations:
+
+- Large Space Agency: Large budget, extensive existing responsibilities.
+- Old Space Agency: Experienced crews and engineers; slower adoption of new technologies and penalties when restructuring or significantly changing processes.
+- Crewed Missions Specialization: Easier and safer crewed missions but significantly increased difficulty and cost for autonomous missions.
```

Comment:
Easier missions in that they are offered easier missions or their crews have experience or bonuses that'd apply to them?